### DISCUSSION TITLE
Could or would it be possible to use a flask in snakeware-os?

### DISCUSSION THREAD

**Discussion by ghost** [pallets/flask#4820] [Q&A] [answered]
Hi all. 

I would like to know if it is possible to use the flask project in the snakeware operating system. 

So, could or would it be possible to use a flask in [snakeware](https://github.com/joshiemoore/snakeware)?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
It's not clear what you are asking for, flask is a normal wsgi Web framework, snake is a entirely custom Linux system

If there is virtualenv and a terminal then it's likely possible to use flask

Anything else is on you as its a relatively unknown linuy distribution that's certainly not in the default set of things to check

**Reply by ghost** [reply]
Hi RonnyPfannschmidt. 

I would like to create a desktop environment made of python. To make this idea possible, I initially thought about adding python as a userspace. But after some research on GitHub I found that there was a project called snakeware - which is based on linux and have python as a linux userspace (which was something I would have to do from scratch and configure). In this sense, my initial work has already been done, that is, I have a distro with linux and python as the userspace of this distribution.

However, I now have the following problem: - I was wondering if the flask community would be interested in snakeware . Whereas I have half the work conceptually to develop(note: I consider the flask to be a great tool and I use it a lot in various types of projects). My question here would be whether the flask community has ever considered porting the flask to snakeware.

If there is this interest in developing flask for snakeware, it would be of great help to me and maybe to the project(snakeware), which I'm really enjoying for the purpose I'm thinking of developing, which is the python desktop environment in snakeware.

**Reply by RonnyPfannschmidt** [reply]
If snakeware is just python on Linux, there is no porting to be done

Else i believe it is unlikely to find interest in the core developers and is possibly considered as unreasonable maintenance cost
